### ISSUE TITLE
124p 데이터의 정규화를 통한 분석 성능 개선하기

### ISSUE THREAD

**Issue by hjkkkl** [hjkkkl/machine-learning#13]
from skleran.preprocessing import norlalize
n_X = normalize(X, axis = 0) #정규화 0번  축 기준으로 실시

nXtrain, nXtest, y_train, y_test = train_test_split(n_X, y, test_size=0.2)
lin_model.fit(nXtrain, y_train)

**Comment by hjkkkl** [comment]
![KakaoTalk_20241002_101515213](https://github.com/user-attachments/assets/07d851fd-f758-4744-9c35-fda5cbd7b975)
